From 86d64630bc8f38709c834b65b589449b7c8a5904 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sun, 12 Aug 2018 00:49:50 +0300
Subject: [PATCH] BTL [Gaudi]: specs got new definitions FIXME !!!!

Just commented out undefined symbols
Need to revisit whole flow for Gaudi
---

diff --git a/board/habanalabs/common/btl/pcie_phy.s b/board/habanalabs/common/btl/pcie_phy.s
index 08000b4..38ee3e5 100644
--- a/board/habanalabs/common/btl/pcie_phy.s
+++ b/board/habanalabs/common/btl/pcie_phy.s
@@ -19,8 +19,10 @@
 /* Select indirect access to CR PARA I/F instead of PRBI */
 cfg mmPCIE_PHY_CR_PARA_CFG, 0x00000001
 
+#ifdef mmPCIE_PHY_MEM_WR_MC
 /* Errata: disable unused design that cause PCIe phy SRAM DERR */
 cfg mmPCIE_PHY_MEM_WR_MC, 0x0
+#endif
 
 /* Adjust the incorrect PHY protocol0 defaults */
 cfg mmPCIE_PHY_PROTOCOL0_EXT_MPLLA_BANDWIDTH, 0x0000A067
diff --git a/board/habanalabs/common/btl/primary.S b/board/habanalabs/common/btl/primary.S
index 9a7e7ef..c5a36b2 100644
--- a/board/habanalabs/common/btl/primary.S
+++ b/board/habanalabs/common/btl/primary.S
@@ -10,14 +10,18 @@
 /* DUMMY */
 nop
 
+#ifdef mmPSOC_GLOBAL_CONF_WARM_REBOOT
 /* Set CPU_BOOT_STATUS_IN_BTL */
 cfg mmPSOC_GLOBAL_CONF_WARM_REBOOT, 0x4
+#endif
 
 /* To avoid false PRSTN configure pad as Schmitt trigger */
 cfg mmPSOC_GLOBAL_CONF_PAD_3V3_CFG_10, 0x41
 
+#ifdef mmPSOC_GLOBAL_CONF_PRSTN_RST_CFG
 /* Configure PRSTN to reset entire SoC */
 cfg mmPSOC_GLOBAL_CONF_PRSTN_RST_CFG, 0x7FFFFF
+#endif
 
 /* Clear masking of PRSTN from SW reset */
 cfg mmPSOC_GLOBAL_CONF_PRSTN_MASK, 0x0
